docs/migration-guides/1.15-1.16.md

```diff
@@ -0,0 +1,10 @@
+#Migrate from 1.15 to 1.16
+
+##Istio
+
+Kyma 1.16 comes with a new Istio version: 1.5.10
+All Istio-related objects in Kyma are updated, but the migration may be necessary for the Istio objects created manually in your cluster.
```

Comment:
"may be" or "is"?
```suggestion
All Istio-related objects in Kyma are updated, but the migration is necessary for the Istio objects created manually in your cluster.
```

Reply:
it depends on the object. Most of them will work, but some don't. Since we don't know, I'd leave "may be" here.

Reply:
As a comment: Istio deprecates many objects, but I'd say "most" of them will still work in 1.15. They stop working in 1.16/1.17 and so on. But the list is really long and I don't see any reason to duplicate Istio's docs here....